src/containers/shedule/Schedule.jsx

```diff
@@ -0,0 +1,88 @@
+import React, { useContext } from "react";
+import styled from "styled-components/macro";
+import { useCookies } from "react-cookie";
+import MetaData from "../../components/metadata/MetaData";
+import { MixcloudWidgetContext } from "../../contexts/MixcloudWidgetContext";
+import { WidgetMarginStyles, PagePaddingStyles } from "../../consts/Styles";
+import useScheduleData from "../../hooks/useScheduleData";
+import ScheduleItem from "../../components/schedule/schedule-item/ScheduleItem";
+import { getShowInPrismic } from "../../helpers/PrismicHelper";
+import Colors from "../../consts/Colors";
+import moment from "moment";
+
+const Schedule = ({ residentsData, currentShow }) => {
+  const { fiveDayAirTimeSchedule } = useScheduleData();
+  const { mixcloudWidgetHtml } = useContext(MixcloudWidgetContext);
+  const [cookies] = useCookies(["ehfm"]);
+
+  return (
+    <>
+      <MetaData
+        title="Schedule | EHFM"
+        description="See what we are about to play next."
+        url="https://www.ehfm.live/schedule"
+      />
+      <Wrapper
+        mixcloudWidgetHtml={mixcloudWidgetHtml}
+        cookiesBannerShowing={!cookies.ehfm}
+      >
+        <ScheduleData>
+          {fiveDayAirTimeSchedule && residentsData
+            ? Object.keys(fiveDayAirTimeSchedule).map((day, i) => {
+                return (
+                  <>
+                    <Name key={`day-${day}-${i}`}>
+                      {fiveDayAirTimeSchedule[day].date.split("NEXT").pop()}
```

Comment:
Might help readability by assigning `fiveDayAirTimeSchedule[day]` to a variable?

Reply:
Good spot - [fixed](https://github.com/RichardPK/eh-fm/pull/45/commits/6f074cd5ee5ec1d2b1672b91f75f9714bb70f583)